src/System.Management.Automation/engine/Utils.cs

```diff
@@ -324,11 +285,7 @@ private static string[] GetProductFolderDirectories()
                 // TODO: #1184 will resolve this work-around
                 // Side-by-side versions of PowerShell use modules from their application base, not
                 // the system installation path.
-#if CORECLR
                 progFileDir = Path.Combine(appBase, "Modules");
-#else
-                progFileDir = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ProgramFiles), "WindowsPowerShell", "Modules");
```

Comment:
Don't we want load Windows PowerShell Modules from %ProgramFiles%\WindowsPowerShell? Currently we use `PSModulePath` as first step solution.